packages/@react-aria/disclosure/test/useDisclosure.test.ts

```diff
@@ -80,6 +80,35 @@ describe('useDisclosure', () => {
     expect(result.current.state.isExpanded).toBe(true);
   });
 
+  it('should not expand or collapse on repeat keydown event', () => {
+    let {result} = renderHook(() => {
+      let state = useDisclosureState({});
+      let disclosure = useDisclosure({}, state, ref);
+      return {state, disclosure};
+    });
+
+    let preventDefault = jest.fn();
+    let event = (e: Partial<KeyboardEvent>) => ({...e, preventDefault} as KeyboardEvent);
+
+    act(() => {
+      result.current.disclosure.buttonProps.onKeyDown?.(event({key: 'Enter', preventDefault}) as KeyboardEvent);
+      result.current.disclosure.buttonProps.onKeyDown?.(event({key: 'Enter', preventDefault, repeat: true}) as KeyboardEvent);
+    });
```

Comment:
```suggestion
    act(() => {
      result.current.disclosure.buttonProps.onKeyDown?.(event({key: 'Enter', preventDefault}) as KeyboardEvent);
    });
    act(() => {
      result.current.disclosure.buttonProps.onKeyDown?.(event({key: 'Enter', preventDefault, repeat: true}) as KeyboardEvent);
    });
```
technically there's enough time between repeat events that each one would be a full state update cycle apart, therefor it should be separated into two acts

same thing below